.gitattributes

```diff
@@ -50,7 +50,7 @@
 *.svg               text        eol=lf
 *.targets           text        eol=lf
 *.tt                text        eol=lf
-*.ttinclude         text        eol=lf
+*.ttinclude         text        eol=crlf
```

Comment:
for some reason `ttinclude` files stop working with `lf` file endings.

Reply:
Yeah, did some reading. No workaround I can find.